wordpress-mobile/wellsql
# Publish to s3

This PR adds support for publishing the library artifacts to our S3 maven using Buildkite. It follows the same approach we used in several other projects. https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2083 was the first project this setup was utilized which contains explanations for some of the decisions made.

---

One small modification it makes to the previous maven structure is that the `wellsql-processor` will no longer be fetched from `org.wordpress:wellsql-processor` but instead will be fetched from `org.wordpress.wellsql:wellsql-processor`. This is so that the library follows our typical maven structure.

The PR also updates `README` to reflect the changes in publishing. I intend to create the tag `1.7.0` once this PR lands, so I used that version in the `README`.

**To test:**
https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2249 can be used to verify the publishing changes.
